integration-tests/src/main.rs

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::Path;
+use std::process::Command;
+
+/// This test runs the payload in the SGX keep using the SGX shim.
+#[cfg_attr(not(has_sgx), ignore)]
+fn sgx_payload() {
+    // Define directories
+    let wksp_root = Path::new(env!("CARGO_MANIFEST_DIR"))
+        .parent()
+        .unwrap()
+        .to_str()
+        .unwrap();
```

Comment:
I think if we remove the `.to_str()`, the following lines of code could be more like:

```rust
let sgx_keep_dir = wksp_root.join("enarx-keep-sgx/target/");
let payload_dir = wksp_root.join("payload/target/x86_64-unknown-linux-musl/");
let sgx_shim_dir = wksp_root.join("enarx-keep-sgx-shim/target/");
```

this would require removing some of the later `.to_owned()` below.